Dockerfile.latest-njs

```diff
@@ -6,7 +6,7 @@ FROM nginx-s3-gateway
 RUN set -eux \
     export DEBIAN_FRONTEND=noninteractive;  \
     apt-get update -qq; \
-    apt-get install --no-install-recommends --no-install-suggests --yes make gcc libc6-dev curl expect libpcre2-dev libpcre3-dev libedit-dev libreadline-dev libssl-dev libpcre2-posix2 libxml2-dev libxslt1-dev zlib1g-dev; \
+    apt-get install --no-install-recommends --no-install-suggests --yes make gcc libc6-dev curl expect libpcre2-dev libpcre3-dev libedit-dev libreadline-dev libssl-dev libpcre2-posix3 libxml2-dev libxslt1-dev zlib1g-dev; \
```

Comment:
**Annotation**:
This change was necessary due to the `libpcre2-posix2` library being unavailable in debian bookworm on which the official nginx image is based.